testsuite/.rubocop_todo.yml

```diff
@@ -258,6 +259,8 @@ Metrics/AbcSize:
 # ExcludedMethods: refine
 Metrics/BlockLength:
   Max: 26
+  Exclude:
+    - 'features/step_definitions/retail_steps.rb'
```

Comment:
I guess we can't refactor the code to don't break this rule?

Reply:
Refactor the code, no, we already tried that and it was a mess.

We could re-introduce the individual `# rubocop:disable Metrics/BlockLength` tags, but I don't see that as much better as changing it for the whole file.